extensions/2.0/Khronos/KHR_materials_transmission/README.md

```diff
@@ -0,0 +1,212 @@
+</figure>
+
+The `KHR_materials_transmission` extension provides a way to define glTF 2.0 materials that are transparent to light in a physically plausible way. That is, it enables the creation of transparent materials that absorb, reflect and transmit light depending on the incident angle and the wavelength of light. Common uses cases for thin-surface transmissive materials include plastics and glass.
+
+This extension aims to address the simplest and most common use cases for optical transparency: infinitely-thin materials with no refraction, scattering, or dispersion. Dealing exclusively with “thin” materials (i.e. materials where only the surface is considered and not the volume) allows many simplifications to be made when calculating things like refraction and absorption.
+
+## Extending Materials
+
+A transparent material is defined by adding the `KHR_materials_transmission` extension to any glTF material. When present, the extension indicates that a material should be rendered as a transparent surface and be blended as defined in the following spec. Note that, as rendering transparent objects presents many difficult-to-solve issues with primitive ordering, this extension does not dictate rendering algorithms.
+
+```json
+materials: [
+  {
+    "extensions": {
+       "KHR_materials_transmission": {
+         "transmissionFactor": 0.8,
+         "transmissionTexture": 0
+       }
+    }
+  }
+]
+```
+
+## Properties 
+Only two properties are introduced with this extension and combine to describe a single value; the percentage of light that is transmitted through the surface of the material. These properties work together with the existing properties of the material to define the way light is modified as it passes through the substance. 
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**transmissionFactor** | `number` | The base percentage of light that is transmitted through the surface.| No, Default: `0.0 ` |
+|**transmissionTexture** | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo) | A texture that defines the transmission percentage of the surface, stored in the `R` channel. This will be multiplied by `transmissionFactor`. | No |
+
+### transmissionFactor 
+The amount of light that is transmitted by the surface rather than diffusely re-emitted. This is a percentage of all the light that penetrates a surface (i.e. isn’t specularly reflected) rather than a percentage of the total light that hits a surface. A value of 1.0 means that 100% of the light that penetrates the surface is transmitted through.
+
+<figure>
+  <img src="./figures/ConstantTransmission.png"/>
+<figcaption><em>baseColor or yellow with transmissionFactor = 1.0.</em></figcaption>
```

Comment:
I think "baseColor of" is meant instead of "baseColor or". 

I also noticed that the example model specifies a magenta baseColorFactor for the model that appears yellowish, which is presumably due to the phrasing in this spec that says:

    rather than absorbed light, we can talk about its inverse: transmitted light

So, I think this needs to be clarified.  Should we subtract baseColor from (1,1,1) ?  Should the caption really say "magenta"?

Reply:
Good catch. Thank you.
It's fixed now.